File: VB/ScrollPictureEdit/Form1.vb
Imports System
Imports System.Data
Imports System.Drawing
Imports System.Windows.Forms
Namespace nsScrollPictureEdit
    Partial Public Class Form1
        Inherits Form
        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
            GridControl1.DataSource = CreateTable(3)
            GridView1.RowHeight = 100

            GridView1.Columns("Image").ColumnEdit = New RepositoryItemScrollPictureEdit()
            CType(GridView1.Columns("Image").ColumnEdit, RepositoryItemScrollPictureEdit).Scrollable = True
            CType(GridView1.Columns("Image").ColumnEdit, RepositoryItemScrollPictureEdit).PictureAlignment = ContentAlignment.TopLeft
        End Sub

        Private Function CreateTable(ByVal RowCount As Integer) As DataTable
            Dim tbl As New DataTable()
            tbl.Columns.Add("Name", GetType(String))
            tbl.Columns.Add("Image", GetType(Image))

            For i As Integer = 0 To RowCount - 1
                tbl.Rows.Add(New Object() {String.Format("Name{0}", i), Resources.Image})
            Next i

            Return tbl
        End Function
    End Class
End Namespace

File: CS/ScrollPictureEdit/Form1.cs
using System;
using System.Data;
using System.Drawing;
using System.Windows.Forms;
using nsScrollPictureEdit.Properties;

namespace nsScrollPictureEdit
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        
        private void Form1_Load(object sender, EventArgs e)
        {
            gridControl1.DataSource = CreateTable(3);
            gridView1.RowHeight = 100;

            gridView1.Columns["Image"].ColumnEdit = new RepositoryItemScrollPictureEdit();
            ((RepositoryItemScrollPictureEdit)gridView1.Columns["Image"].ColumnEdit).Scrollable = true;
            ((RepositoryItemScrollPictureEdit)gridView1.Columns["Image"].ColumnEdit).PictureAlignment = ContentAlignment.TopLeft;
        }

        private DataTable CreateTable(int RowCount)
        {
            DataTable tbl = new DataTable();
            tbl.Columns.Add("Name", typeof(string));
            tbl.Columns.Add("Image", typeof(Image));

            for ( int i = 0; i < RowCount; i++ )
                tbl.Rows.Add(new object[] { String.Format("Name{0}", i), Resources.Image });

            return tbl;
        }
    }
}